upgrade_test.py

```diff
@@ -252,6 +252,14 @@ def test_rolling_upgrade(self):
         well, upgrade all nodes to new version in the end.
         """
 
+        self.log.info('Populate DB with many types of tables and data')
+        self.fill_db_data()
+        self.log.info('Run some Queries to verify data BEFORE UPGRADE')
+        self.verify_db_data()
+        self.clean_db_data()
```

Comment:
We have delete queries in verify_db_data(), the data will be changed in verifying.
It will be wrong in repeat verify after upgrade. So I clean the data and refill before upgrade.